client/coral-admin/src/components/UserDetail.js

```diff
@@ -100,36 +87,90 @@ export default class UserDetail extends React.Component {
     );
   }
 
+  getActionMenuLabel() {
+    const {root: {user}} = this.props;
+
+    if (isBanned(user)) {
+      return 'Banned';
+    } else if (isSuspended(user)) {
+      return 'Suspended';
+    }
+
+    return '';
+  }
+
   renderLoaded() {
     const {
       data,
       root,
       root: {
+        me,
         user,
         totalComments,
         rejectedComments,
-        comments: {nodes, hasNextPage}
+        comments: {
+          nodes,
+        }
       },
       activeTab,
       selectedCommentIds,
       toggleSelect,
       hideUserDetail,
       viewUserDetail,
       loadMore,
-      toggleSelectAll
+      toggleSelectAll,
+      unBanUser,
+      unSuspendUser,
```

Comment:
I'd call them `unbanUser` and `unsuspendUser` as `unban` and `unsuspend` is one word.